### ISSUE TITLE
[FR] add hook support

### ISSUE THREAD

**Issue by ComfortablyCoding** [ComfortablyCoding/strapi-plugin-transformer#54]
### Request
Add pre and post transform hooks for users to utilize for any additional modifications they wish to make.

### Example
```js
module.exports = ({ env }) => ({
  // ..
 'transformer': {
    enabled: true,
    config: {
      // ...
      hooks :{
       'preResponseTransform': (ctx)=>{}
       'postResponseTransform': (ctx)=>{}
      }
      // ...
    }
  },
  // ..
});
```

P.S. Brought up in #46.

**Comment by raphaeldas** [comment]
Great! Thank you!

**Comment by ComfortablyCoding** [comment]
Yoir welcome, it should be available in v2.2.0
